src/test/java/co/rsk/bitcoinj/script/RedeemScriptValidatorTest.java

```diff
@@ -24,10 +24,37 @@ public class RedeemScriptValidatorTest {
     private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(220));
     private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(330));
 
+    private Script standardRedeemScript;
+    private Script flyoverStandardRedeemScript;
+    private Script nonstandardErpRedeemScript;
+    private Script flyoverNonStandardErpRedeemScript;
+    private Script p2shErpRedeemScript;
+    private Script flyoverP2shErpRedeemScript;
+    private Script nonStandardErpTestnetRedeemScript;
+
     @Before
     public void setUp() {
```

Comment:
I noticed you can refactor the variables in the `setUp` method into constant variables, provided they don’t need to be recreated for each test run